### ISSUE TITLE
Panel focus outline doesn't work correctly with iframes

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#12832]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 2023.05.0-daily+52
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem

- Turn on [Tools / Global Options / Accessibility / "Highlight focused panel"]
- This causes a highlight to be shown around the IDE region (Console, Editor, Tabset1, Tabset2, etc.) that currently contains keyboard focus
- With focus in the Console (<kbd>Ctrl+2</kbd>, note the focus outline
- Switch to Help pane via <kbd>Ctrl+3</kbd> (or use <kbd>F6</kbd> or <kbd>Shift+F6</kbd> to move between panes)

### Describe the problem in detail
- Focus is placed in the help pane's content, which is an iframe
- The technique for detecting where focus is doesn't handle iframes, and the focus outline remains on Console region

### Describe the behavior you expected
- the correct region should be highlighted even if focus is in an iframe inside that region

### Notes

* PR that added focus outline option: https://github.com/rstudio/rstudio/pull/7881

* The code that tries to detect focus location is here: https://github.com/rstudio/rstudio/blob/59c3bc594478395db19d8b952024221791f46fe9/src/gwt/src/org/rstudio/studio/client/workbench/ui/PaneManager.java#L497-L526

* see also #10215, #7888

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
This is a low priority. The feature is off by default, and nobody has reported this particular problem. Moving out of chocolate cosmos.
